src/routes/Routes.tsx

```diff
@@ -79,6 +79,9 @@ const ElvaYBPractice = React.lazy(
 const SkDashboard = React.lazy(
   () => import(/* webpackChunckName: "SkDashboard" */ '../pages/FrontendMentor/SkDashboard'),
 )
+const MaximWpWebsite = React.lazy(
+  () => import(/* webpackChunckName: "Maximweipin" */ '../pages/FrontendMentor/Maximweipin'),
```

Comment:
try to keep to the same name, so webpackChunckName should be MaximWpWebsite instead of Maximweipin